<?xml version="1.0"?>

<launch>

  <node pkg="simple_image_processing" type="rgb_hough_node" name="rgb_hough" output="screen" >
    <remap from="image_in" to="/usb_cam/image_raw" />
  </node>

  <!-- Decompress image from bag file -->
  <node pkg="image_transport" type="republish" name="republish" args="in:=/image compressed out:=/raw_image" />
</launch>
